regexps.json

```diff
@@ -0,0 +1,5 @@
+{
+    "re1": "([0-9]+) (day?|week?|month?|year?)(:?s)? ago",
+    "re2": "(?:last|previous) (Mon|Tues|Wednes|Thurs|Fri|Satur|Sun)(?:day)?",
```

Comment:
@light94 : I think we should use the standard three/four letter days. `mon, tue, Wed, Thurs, Fri, sat, sun`. (A couple of days need to change)